### PR TITLE
[full-ci] Retry Postprocessing

### PR DESCRIPTION
**Pull request by kobergj**
Adds a ctl command to retry failed postprocessing by upload id

Fixes https://github.com/owncloud/ocis/issues/6495

### CODE REVIEW

File: services/postprocessing/README.md [hunk 1]
```diff
@@ -60,3 +60,19 @@ When setting a custom postprocessing step (eg. `"customstep"`) the postprocessin
...
 
 See the [cs3 org](https://github.com/cs3org/reva/blob/edge/pkg/events/postprocessing.go) for up-to-date information of reserved step names and event definitions.
+
+## CLI Commands
+
+### Resume Postprocessing
+
+As mentioned before, if postprocessing fails in one step due to an unforseen error, current uploads will not be retried automatically. A system admin can instead run a cli command to retry the failed upload. This is a two step process.
+
+- first find the uploadid of the failed upload
```
Comment (mmattel) [inline services/postprocessing/README.md:70]:
```suggestion
-   First find the upload ID of the failed upload.
```

File: services/postprocessing/README.md [hunk 2]
```diff
@@ -60,3 +60,19 @@ When setting a custom postprocessing step (eg. `"customstep"`) the postprocessin
...
 See the [cs3 org](https://github.com/cs3org/reva/blob/edge/pkg/events/postprocessing.go) for up-to-date information of reserved step names and event definitions.
+
+## CLI Commands
+
+### Resume Postprocessing
+
+If postprocessing fails in one step due to an unforseen error, current uploads will not be retried automatically. A system admin can instead run a CLI command to retry the failed upload which is a two step process:
+
+-   First find the upload ID of the failed upload.
+```
```
Comment (mmattel) [inline services/postprocessing/README.md:71]: I cant make a suggestion...
You need to add post the three ticks the term `bash` to silence the remaining codacy complain.
The same on the next cli command.

**Comment by kobergj** [comment]
@wkloucek yes this is possible. It would need a little scripting as one needs to get the list of unfinished uploads via `ocis storage-users uploads list` first. But no magic.

I guess it is more a question to PMs if we want it like this.

**Review by rhafer — APPROVED** [review]
:+1:

**Pull request merged**